Sami/Parser/NodeVisitor.php

```diff
@@ -151,8 +152,12 @@ protected function addMethod(ClassMethodNode $node)
             if ($param->default) {
                 $parameter->setDefault($this->context->getPrettyPrinter()->prettyPrintExpr($param->default));
             }
-            if ((string) $param->type) {
-                $parameter->setHint($this->resolveHint(array(array((string) $param->type, false))));
+            if ($type = (string) $param->type) {
+                if( $param->type instanceof FullyQualified && strpos($type, '\\') !== 0 ) {
```

Comment:
> Can you remove the spaces after '(' and before ')' and add a space after if?

Usually `fabpot.io` detects any CS issues and offers quick fix for them. Maybe it's broken.